features/stream/src/main/java/com/m3u/features/stream/PlayerActivity.kt

```diff
@@ -52,7 +53,8 @@ class PlayerActivity : ComponentActivity() {
             ) {
                 StreamRoute(
                     onBackPressed = { finish() },
-                    viewModel = viewModel
+                    viewModel = viewModel,
+                    getProgrammeCurrently = { channelId -> viewModel.getProgrammeCurrently(channelId) },
```

Comment:
You should pass the param into the Route function. because its implementation seems is just invoking the function in viewmodel